packages/select/src/components/multi-select/multiSelect2.tsx

```diff
@@ -203,6 +204,7 @@ export class MultiSelect2<T> extends AbstractPureComponent2<MultiSelect2Props<T>
                     <TagInput
                         placeholder={placeholder}
                         {...tagInputProps}
+                        aria-expanded={this.state.isOpen}
```

Comment:
this needs to go on the wrapping div... see how we are setting another aria attribute here:
https://github.com/palantir/blueprint/blob/7ea317e054565a23dc4407f7feb5bef4553c195c/packages/popover2/src/popover2.tsx#L344